lib/utils/xml-helper.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
```

Comment:
Please add test coverage for this file.
Should be a relatively simple test with a mock on 'xmlbuilder'